spec/titles/title_from_245_subfield_a_b_spec.rb

```diff
@@ -30,7 +30,7 @@
       expect(graph.query(TITLE_PROPERTY_245_QUERY).to_tsv).to include("Slippery noodles :")
     end
     it 'has a bf:mainTitle property' do
-      expect(graph.query(TITLE_PROPERTY_245_QUERY).to_tsv).to include("<http://bib.ld4l.org/ontology/MainTitleElement>")
+      expect(graph.query(TITLE_PROPERTY_245_QUERY).to_tsv).to include("<http://data.ld4l.org/ontology/MainTitleElement>")
```

Comment:
So this namespace is actually a configuration detail within the bib2lod project, and I have it configures as: `src/main/resources/config.json:    "local_namespace": "http://data.ld4l.org/stanford/"`.

I can change it to `bibliotek-o.org` but I should also make a note of this in the README because the configuration and the expectation need to match, and the test writer needs to know this. Stay tuned for a follow up PR.